parser/prism/Translator.cc

```diff
@@ -1212,7 +1212,11 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             // constant assignment errors
             // TODO: We will improve this in the future when we handle more errored cases
             for (auto &error : parser.parseErrors) {
-                reportError(translateLoc(error.location), error.message);
+                // EOF error is always pointed to the very last line of the file, which can't be expressed in Sorbet's
+                // error comments
```

Comment:
Could we provide an alternative error location that makes this work?

E.g. to match this behaviour: https://sorbet.run/#%23%20typed%3A%20false%0A%0A%22missing%20close%20quote